brand/admin.py

```diff
@@ -114,7 +114,7 @@ def associated_contacts(self, obj):
             {"fields": (("from_the_website",), ("institution_type", "institution_credentials"))},
         ),
         ("CMS", {"fields": (("subtitle",), ("header",), ("summary",), ("details",))}),
-        ("Harvest Data", {"fields": ("feature_yaml", "feature_refresh_date")}),
+        ("Harvest Data", {"fields": ("feature_override", "feature_yaml", "feature_refresh_date")}),
```

Comment:
Please hide the feature_override field for now. This is the next ticket, and will need to be a YAML editor anyway